ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/mapper/claimantresponse/ClaimantResponseMapper.java

```diff
@@ -0,0 +1,90 @@
+package uk.gov.hmcts.cmc.ccd.domain.mapper.claimantresponse;
```

Comment:
please move this into top mapper package

Reply:
please check my last commit for this PR.